docs/docsite/rst/filter_guide.rst

```diff
@@ -247,55 +248,34 @@ This produces:
 Merging lists of dictionaries
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-If you have two lists of dictionaries and want to combine them into a list of merged dictionaries, where two dictionaries are merged if they coincide in one attribute, you can use the ``lists_mergeby`` filter.
+If you have two or more lists of dictionaries and want to combine them into a list of merged dictionaries, where the dictionaries are merged by an attribute, you can use the ``lists_mergeby`` filter.
 
-.. code-block:: yaml+jinja
+.. note:: The output of the examples in this section use the yaml callback plugin. Quoting: "Ansible output that can be quite a bit easier to read than the default JSON formatting." See ``shell> ansible-doc -t callback yaml``
 
-    - name: Merge two lists by common attribute 'name'
-      debug:
-        var: list1 | community.general.lists_mergeby(list2, 'name')
-      vars:
-        list1:
-          - name: foo
-            extra: true
-          - name: bar
-            extra: false
-          - name: meh
-            extra: true
-        list2:
-          - name: foo
-            path: /foo
-          - name: baz
-            path: /bazzz
+In the example below the lists are merged by the attribute *name*
+
+.. literalinclude:: ../examples/lists_mergeby/example-001.yml
+   :language: yaml
```

Comment:
The file is not pure YAML. Generally we always use `yaml+jinja` for something that can potentially contain Jinja2.
```suggestion
   :language: yaml+jinja
```